docs/package.json

```diff
@@ -21,6 +21,7 @@
 		"@mdx-js/react": "^3.0.0",
 		"classnames": "^2.3.2",
 		"clsx": "^2.0.0",
+		"docusaurus": "^1.14.7",
```

Comment:
This is a deprecated version. The new versions start with `@docusaurus/[sub-package-name]` (https://www.npmjs.com/search?q=%40docusaurus). And we had the needed sub-packages already added.
```suggestion
```